README.md

```diff
@@ -8,50 +8,80 @@ Creusot is a tool for *deductive verification* of Rust code. It allows you to an
 
 Creusot works by translating Rust code to WhyML the verification and specification language of ![Why3](https://why3.lri.fr). Users can then leverage the full power of Why3 to (semi)-automatically discharge the verification conditions!
 
-**Note**: :warning: I am developing this in the context of my PhD thesis, the software quality is commensurate. :warning:
+See [ARCHITECTURE.md](ARCHITECTURE.md) for technical details!
 
-# Example programs that have been proven:
+**Note**: :warning: This tool is mainly for research purposes now. Please tolerate the software quality! :warning:
+
+# Examples of Verification
 
 - [Mutably indexing a linked list](creusot/tests/should_succeed/list_index_mut.rs)
 - [Zeroing out a list](creusot/tests/should_succeed/all_zero.rs)
-- [Binary Search](creusot/tests/should_succeed/binary_search.rs)
-
+- [Binary search](creusot/tests/should_succeed/binary_search.rs)
 
+More examples are found in [creusot/tests/should_succeed](creusot/tests/should_succeed).
 
 # Installing
 
-0. Install Rust using `rustup` to manage toolchains
-1. Clone the repository
-2. Install the Rust compiler libraries: `rustup component add rustc-dev`
-3. (Optional, Recommended) Install the Rust compiler sources: `rustup component add rustc-src`
-4. Run `cargo build`
-
-# Generating MLCFG outputs
-
-
+0. Clone the [creusot](https://github.com/xldenis/creusot/) repo, set the current directory to that folder
+1. Set up Rust
+    - [Install `rustup`](https://www.rust-lang.org/tools/install), to get suitable Rust toolchains
+    - Make sure you are at the repo dir (so that rustup can load [`rust-toolchain`](rust-toolchain))
+    - Install the basic toolchain: `$ rustup component add rustc-dev`
```

Comment:
no need! this is handled in the `rust-toolchain` file. All you need to do is `cargo build` and it will fetch all the correct stuff.

Reply:
You don't need the `rustup component add` bit, it comes for free.